Getting Started/v1-12-2 Overview.rst

```diff
@@ -36,6 +37,20 @@ Features Outlined in this Document
 * Fix DatePicker on client side
 * Add Custom Forms to Ticket Filter Data
 
+Recipients Field for Departments
+--------------------------------
+
+Please note, we had a bug fix in this release that corrected the behavior of the Recipients field for Departments. It was incorrectly sending out alerts even if Recipients was set to ‘No one’. If you want to change the behavior of this field, you can change it by going to:
+
+Admin Panel | Agents Tab | Departments | Select the desired department
+
+.. image:: ../_static/images/111overview_dept_recip1.png
+  :alt: Email Template Name
+
+The Recipients field is under Alerts and Notices on the form. If it is set to ‘No one (disable Alerts and Notices)’, Agents will NOT receive an alert when a Ticket is created in this Department. If it is set to ‘Department members only’, only Agents with this Department as a primary Department will receive the alert. If it is set to ‘Department and extended access members’, all Agents with access to this Department will receive the alert.
+
+.. image:: ../_static/images/111overview_dept_recip2.png
+  :alt: Email Template Name
```

Comment:
You named both of the image `:alt:` the same, is this intended?

Reply:
Fixed. Sorry about that... They're both Recipients Field for Departments now. Should be fine for them both to be the same since that's just what you would see if the page doesn't load the images.

Reply:
Think of it more in terms of people with visual disabilities. They will use a screenreader to read the page, and images are read using their `alt` tag value. I like to give images a unique `alt` name that describes what the image is, suggests, or contains. So for example, your first image I would put `Select Desired Department` and for the second image I would put `Configure Alert Recipients`. Just some thought for the future. 🙂